backend/Utils/Edna.Utils.Http/Edna.Utils.Http/EdnaExternalHttpHandler.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.Net.Http;
+using System.Security.Cryptography.X509Certificates;
+using System.Net.Security;
+
+namespace Edna.Utils.Http
+{
+    public class EdnaExternalHttpHandler: HttpClientHandler
+    {
+        public static string Name = nameof(EdnaExternalHttpHandler);
+        public EdnaExternalHttpHandler() : base()
+        {
+            ServerCertificateCustomValidationCallback = PerformX509Valiation;
+        }
+
+        public bool PerformX509Valiation ( HttpRequestMessage req, X509Certificate2 cert, X509Chain chain, SslPolicyErrors err )
+        {
+            return (err == SslPolicyErrors.None) && chain.Build(cert);
```

Comment:
I might need some explanation or pointers on what this is trying to achieve and how it is doing that.

Reply:
The following links should help in explaining the above invocation:
- `X509Chain` `build` method: https://docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.x509certificates.x509chain.build?view=netcore-3.1 
- `X509VerificationFlags` used by the above `build` method to verify the certificates in the chain: https://docs.microsoft.com/en-us/dotnet/api/system.security.cryptography.x509certificates.x509verificationflags?view=netcore-3.1 (by default: NoFlag i.e. all the checks are performed)